conda/recipes/meta.yaml

```diff
@@ -8,7 +8,9 @@ source:
 
 build:
   noarch: python
-  script: pip install . -vv
+  script:
+    - export CMAKE_PREFIX_PATH=${PREFIX}/targets/x86_64-linux/lib:$CMAKE_PREFIX_PATH
+    - pip install . -vv
```

Comment:
Just noting that this change was needed for CMake to find `libnvjitlink-static`

Reply:
Added a note